starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,44 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class="top-image">
+    <h1>Apple Pie</h1>
+  </div>
+  <div class="recipe">
+    <p>This was my grandmother's apple pie recipe. I have never seen another one
+    quite like it. It will always be my favorite and has won me several first place
+    prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+    
+    <img src="images/recipe-info.png" alt="recipe information" class="information-image">
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <hr>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+    <h2>Ingredients</h2>
+    <ul class="ingredients-list">
+      <li class="list-odd">- 1 recipe pastry for a 9 inch double crust pie</li>
```

Comment:
There is better way to add the dashes, imagine you have 500 lines and you have to add each one manually. Hint:
```
ul.name-of-the-class > li:before {
  content: "- ";
}
```